externals/sqlite3/src/sqlite3.h

```diff
@@ -913,7 +913,7 @@ struct sqlite3_io_methods {
 ** in order for the database to be readable.  The fourth parameter to
 ** [sqlite3_file_control()] for this opcode should be a pointer to an integer.
 ** That integer is 0 to disable persistent WAL mode or 1 to enable persistent
-** WAL mode.  If the integer is -1, then it is overwritten with the current
+** WAL mode.  If the integer is -1, then it is overridden with the current
```

Comment:
sqlite's an external library. And there's a branch w/ my changes for it anyway, so the sqlite stuff should really be dropped.

If the entire `externals` directory should be skipped, let me know, that's easy to do.